docs/build-and-display-linux.md

```
@@ -0,0 +1,200 @@
+```
+
+そのディスプレイを指定して起動する。
+
+```bash
+DISPLAY=:108 target/debug/nohrs
+```
+
+GPU が無い環境では llvmpipe で描画するため、**起動直後の数秒はウィンドウが黒いまま**
+になる（フォントアトラスや UI 要素の生成待ち）。ログに以下が出れば描画準備完了:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)